README.md

```diff
@@ -23,14 +26,27 @@ Wireless Tool for ITC
 ### File Structure
 
 ```
-├──[wireless-tool]/              # root directory
-|  └──[README.md]                # this file!
+├──[wireless-tool]/ # root directory
+    ├── README.md
+    ├── requirements.txt
+    ├── server
+    │   ├── components
+    │   ├── config
+    │   └── index.py # the root file of the server
+    ├── bin        
+    ├── include
+    └── lib
 ```
 
 For more detailed documentation on our file structure and specific functions in the code, feel free to check the project files themselves.
 
 ## Setup Steps 
-1. Clone repo by running `git clone https://github.com/dali-lab/wireless-tool` in your terminal
+1. Clone the dev branch `git clone https://github.com/dali-lab/wireless-tool.git --branch dev` in your terminal
+2. Move into the root folder `cd wireless-tool`
+4. Install virtualenv `pip3 install virtualenv`
+3. Install everything `virtualenv .env && source .env/bin/activate && pip install -r requirements.txt` (should be executed at root level, the wireless-tool folder)
```

Comment:
``` bash
(.env) Jais-MacBook-Pro:wireless-tool jaismith$ pip install -r requirements.txt
Collecting falcon==2.0.0
  Using cached https://files.pythonhosted.org/packages/4b/89/37b132c6ae7f85c60a1549f9e46109eec40cf0c31f0305faecb7f7bcfceb/falcon-2.0.0-py2.py3-none-any.whl
Collecting gunicorn==20.0.4
  Using cached https://files.pythonhosted.org/packages/69/ca/926f7cd3a2014b16870086b2d0fdc84a9e49473c68a8dff8b57f7c156f43/gunicorn-20.0.4-py2.py3-none-any.whl
Collecting psycopg2==2.8.4
  Using cached https://files.pythonhosted.org/packages/84/d7/6a93c99b5ba4d4d22daa3928b983cec66df4536ca50b22ce5dcac65e4e71/psycopg2-2.8.4.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /Users/jaismith/Documents/Code/wireless-tool/.env/bin/python3.7 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/b2/_jr50r993nz2d7syxws2brjh0000gn/T/pip-install-fg3rbiod/psycopg2/setup.py'"'"'; __file__='"'"'/private/var/folders/b2/_jr50r993nz2d7syxws2brjh0000gn/T/pip-install-fg3rbiod/psycopg2/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/b2/_jr50r993nz2d7syxws2brjh0000gn/T/pip-install-fg3rbiod/psycopg2/pip-egg-info
         cwd: /private/var/folders/b2/_jr50r993nz2d7syxws2brjh0000gn/T/pip-install-fg3rbiod/psycopg2/
    Complete output (23 lines):
    running egg_info
    creating /private/var/folders/b2/_jr50r993nz2d7syxws2brjh0000gn/T/pip-install-fg3rbiod/psycopg2/pip-egg-info/psycopg2.egg-info
    writing /private/var/folders/b2/_jr50r993nz2d7syxws2brjh0000gn/T/pip-install-fg3rbiod/psycopg2/pip-egg-info/psycopg2.egg-info/PKG-INFO
    writing dependency_links to /private/var/folders/b2/_jr50r993nz2d7syxws2brjh0000gn/T/pip-install-fg3rbiod/psycopg2/pip-egg-info/psycopg2.egg-info/dependency_links.txt
    writing top-level names to /private/var/folders/b2/_jr50r993nz2d7syxws2brjh0000gn/T/pip-install-fg3rbiod/psycopg2/pip-egg-info/psycopg2.egg-info/top_level.txt
    writing manifest file '/private/var/folders/b2/_jr50r993nz2d7syxws2brjh0000gn/T/pip-install-fg3rbiod/psycopg2/pip-egg-info/psycopg2.egg-info/SOURCES.txt'
    
    Error: pg_config executable not found.
    
    pg_config is required to build psycopg2 from source.  Please add the directory
    containing pg_config to the $PATH or specify the full executable path with the
    option:
    
        python setup.py build_ext --pg-config /path/to/pg_config build ...
    
    or with the pg_config option in 'setup.cfg'.
    
    If you prefer to avoid building psycopg2 from source, please install the PyPI
    'psycopg2-binary' package instead.
    
    For further information please check the 'doc/src/install.rst' file (also at
    <http://initd.org/psycopg/docs/install.html>).
    
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```